plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/changedpanel/ChangedPanelView.java

```diff
@@ -60,49 +59,68 @@
      * @param files
      *         Map of changed files with their status
      */
-    void viewChangedFilesAsList(@NotNull Map<String, Status> files);
+    void viewChangedFilesAsList(Map<String, Status> files);
```

Comment:
You have two method to show files, as above I mentioned, create enum and only one method, which will consume file list and view mode.